components/button.json

```diff
@@ -41,6 +41,24 @@
         "background": "@theme-button-secondary-normal",
         "text": "@theme-text-primary-disabled",
         "border": "@theme-outline-disabled-normal"
+      },
+      "active": {
```

Comment:
We can't nest active inside secondary, because secondary is an object with the different UI states within it. Can you move this up a level and rename it to something else please?